Code/src/main/java/nl/utwente/viskell/ui/components/DrawWire.java

```diff
@@ -0,0 +1,124 @@
+package nl.utwente.viskell.ui.components;
+
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.geometry.Point2D;
+import javafx.scene.shape.CubicCurve;
+import javafx.scene.transform.Transform;
+import nl.utwente.ewi.caes.tactilefx.control.TactilePane;
+import nl.utwente.viskell.ui.ComponentLoader;
+import nl.utwente.viskell.ui.CustomUIPane;
+
+/**
+ * A DrawWire is a helper class to represent a new incomplete connection is the process of being drawn. 
+ * It is linked to a single Anchor as starting point, and with a second anchor it produces a new Connection.
+ */
+public class DrawWire extends CubicCurve implements ChangeListener<Transform>, ComponentLoader {
+
+    /** The Pane this connection wire is on. */
+    private final CustomUIPane pane;
+
+    /** The Anchor this wire has been initiated from */
+    private final ConnectionAnchor anchor;
+    
+    /**
+     * @param pane The Pane this wire is on.
+     * @param anchor The starting anchor of new wire.
+     */
+    public DrawWire(CustomUIPane pane, ConnectionAnchor anchor) {
+        this.loadFXML("DrawWire");
+        TactilePane.setDraggable(this, false);
+        TactilePane.setGoToForegroundOnContact(this, false);
+        this.setMouseTransparent(true);
+
+        this.pane = pane;
+        this.anchor = anchor;
+        pane.getChildren().add(this);
+        Point2D initPos = pane.sceneToLocal(anchor.localToScene(anchor.getLocalCenter()));
+        this.setFreePosition(initPos);
+        this.invalidateAnchorPosition();
+        anchor.localToSceneTransformProperty().addListener(this);
```

Comment:
Would prefer something like

    anchor.localToSceneTransformProperty().addListener(x -> this.invalidateAnchorPosition());

as this would enable one to drop the ChangeListener interface.